wiki/Installation/id.md

```diff
@@ -1,70 +1,77 @@
-    - Jika anda perlu mendaftar, lihat [Registrasi](/wiki/Registration), lalu masuk.
+3. Saat penginstal berjalan, akan ada waktu sepuluh detik untuk memungkinkan kamu memilih lokasi pemasangan.
+   - Jika kamu ingin mengubahnya, klik pada teks di bawah kata, `Hi!` dan pilih lokasi pemasangan.
+   - osu! akan dipasang berdasarkan lokasi yang kamu pilih saat waktu mundur telah selesai.
+4. osu! akan terbuka sesaat setelah proses pemasangan selesai. osu!direct akan digunakan untuk mengunduh beberapa beatmap pengantar untukmu.
+5. Terakhir osu! akan meminta kamu untuk masuk atau mendaftar.
+   - Jika kamu sudah memiliki akun, silakan masuk.
+   - Jika kamu perlu mendaftar, lihat artikel [Registrasi](/wiki/Registration), lalu masuk.
+
+## Penyelesaian masalah
+
+Jika kamu menggunakan Windows 7 64-bit, kamu mungkin akan mengalami beberapa kendala saat memasang osu!. Kamu bisa coba memasang .NET Framework versi 64-bit, jika program ini tidak disertakan dalam penginstal osu. [Unduh di sini](https://download.microsoft.com/download/2/0/e/20e90413-712f-438c-988e-fdaa79a8ac3d/dotnetfx35.exe)
+
+---
+
+**Jika penginstal osu! memberikan kode kesalahan saat dipasang, kamu dapat mencoba beberapa hal berikut:**
+
+- Coba pasang ulang .NET Framework dan pastikan untuk menggunakan versi terbaru. [Kamu dapat mengunduhnya lewat tautan berikut](https://dotnet.microsoft.com/download)
+- Bersihkan PC kamu - Pindai dengan anti-virus, bersihkan juga registry-nya. Program yang bagus untuk melakukan pembersihan umum adalah CCleaner. Pindai file komputer dan registry kamu. Pemasang osu! bisa saja gagal dijalankan dikarenakan ada entri registry osu! dari salinan osu! yang masih ada dalam sistem kamu.
 
-## Menambahkan beatmapsets
+## Menambahkan beatmapset
 
-*Lihat juga: [Beatmaps](/wiki/Beatmaps) dan [Beatmapsets](/wiki/Glossary#beatmapset)*
+*Lihat juga: [Beatmaps](/wiki/Beatmaps)*
 
 Ada dua cara untuk menambahkan beatmap, melalui situs web atau osu!direct (memerlukan [osu!supporter](/wiki/osu!supporter)).
 
-### Website
+### Situs web
 
-1. Kunjungi [halaman daftar beatmap](https://osu.ppy.sh/beatmapsets) dan masuk ke situs web.
-2. Cari lagu yang akan anda mainkan.
-3. Klik pada ikon unduh untuk menyimpan beatmapset di komputer anda.
+1. Kunjungi [laman daftar beatmap](https://osu.ppy.sh/beatmapsets) dan masuk ke situs web.
+2. Cari lagu yang ingin kamu mainkan.
+3. Klik pada ikon unduh untuk menyimpan beatmapset di komputer kamu.
 4. Cari dan buka file beatmapset, file ini memiliki ekstensi `.osz`.
-    - Sebagai alternatif, anda juga dapat menempatkan file beatmapset ke folder `Songs` pada direktori osu! anda.
-5. osu! secara otomatis akan menangani sisanya untuk anda dan memuat beatmapset.
-    - Jika, karena alasan apa pun, anda tidak dapat melihatnya, tekan `F5` dalam Mode [Solo](/wiki/Interface#song-selection-screen) untuk memproses kembali beatmap anda.
+   - Cara lain, kamu juga dapat menempatkan file beatmapset ke folder `Songs` pada direktori osu! kamu.
```

Comment:
` Ada Cara yang lain, kamu juga ...`

More understandable rather or just delete it for better readability.